contracts/Core.sol

```diff
@@ -259,18 +269,17 @@ contract Core is DBC, Owned, Shares, SafeMath, CoreProtocol {
 
     /// Pre: Universe has been defined
     /// Post: Whether buying and selling of tokens are allowed at given exchange
-    function isWithinKnownUniverse(address onExchange, address sell_which_token, address buy_which_token)
+    function requireIsWithinKnownUniverse(address onExchange, address sell_which_token, address buy_which_token)
```

Comment:
ideally yes! however there is [this one time](https://github.com/melonproject/protocol/blob/c1c3a8d670fbcf2402528eb90e96c9d8e44ac0c2/contracts/Core.sol#L244-L249) where it needs to get (call) the parameteres for it first